src/Tests/Microsoft.Diagnostics.Monitoring.StartupHook.UnitTests/Exceptions/Eventing/ExceptionsEventListener.cs

```diff
@@ -68,6 +68,7 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
                             ToUInt64(eventData.Payload[NameIdentificationEvents.FunctionDescPayloads.FunctionId]),
```

Comment:
Can we add a couple tests (or update existing ones) to validate that the new data is being reported correctly for both exception and stacks?

Reply:
I've updated both the startuphook unit tests and the exceptions functional tests. Let me know if there are other areas where I can add more tests.

Reply:
I'm assuming this PR also updates `/stacks` to report this information so ideally we'd update those tests by updating the following lines:
https://github.com/dotnet/dotnet-monitor/blob/32fa38302fa3293889835062e611ea753b564618/src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/StacksTests.cs#L536
https://github.com/dotnet/dotnet-monitor/blob/32fa38302fa3293889835062e611ea753b564618/src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/StacksTests.cs#L64


If we take a project reference on `Microsoft.Diagnostics.Monitoring.UnitTestApp.dll` we can determine the expected mvid and method tokens for the frames.

Reply:
The mvid and mdtokens show up in the JSON response so I've updated `JsonValidation` instead of `PlainTextValidation`